datumaro/plugins/transforms.py

```diff
@@ -1121,9 +1121,11 @@ class RemoveAnnotations(ItemTransform):
     @staticmethod
     def _parse_id(s):
         full_id = s.split(":")
```

Comment:
```suggestion
        full_id = s.split(":")
        if len(full_id) == 3:
            full_id[-1] = int(full_id[-1])
```

`s` will be string in CLI arguments, so that we need to apply type conversion. It would be better to add a CLI integration test for this.